### DISCUSSION TITLE
LowFER

### DISCUSSION THREAD

**Discussion by 4c3e** [markqvist/Reticulum#62] [Help & Questions]
Anyone here have any experience with running packet radio over [LowFER](https://en.wikipedia.org/wiki/LowFER)? If it's possible to legally send encrypted packets on this bandwidth, it could be an extremely useful radio bridge between communities, with transmission ranges of 1000 miles being noted in the wikipedia article.

I'd love to learn more about transmitting over this band if anyone has any useful references. Unfortunately I don't have any experience with HAM radio so far.

**Comment by OtisByron** [comment]
@4c3e I don't have LowFER experience, but I was just doing a little research after reading your post. It looks like the 160-190kHz LowFER band is provided under FCC Part 15 subsection 217, and I didn't see anything prohibiting encryption.

I also noticed in my brief research that most LowFER radio equipment is homemade, and in my experience most amateur radio transceivers don't transmit that low in frequency. FCC Part 15 does require marketed products to be Part certified, and prevents the marketing of kits that result in equipment that should be Part certified, so even designing new transceiver hardware that is both legal and easily accessible to unlicensed users may be challenging.

That being said this a very interesting topic for the reasons your described, and a valuable piece of the puzzle. Hopefully others with more experience on the topic can offer more information. I plan to dig around on this topic some more, especially on the transceiver hardware side.
